src/docs/src/routes/(docs)/docs/themes/+page.svelte.md

```diff
@@ -143,7 +143,7 @@ module.exports = {
 - <Translate text="The first theme (<code>mytheme</code>) will be the default theme." />
 - <Translate text="<code>dark</code> theme will be the default theme for dark mode." />
 
-<Translate text="In the below example, I have the required colors. All other colors will be generated automatically (Like the color of button when you focus on it or the color of text on a <code>primary</code> button)." />
+<Translate text="In the below example, I have the required default colors. All other colors will be generated automatically (Like the color of button when you focus on it or the color of text on a <code>primary</code> button)." />
```

Comment:
It's not default. the colors in example are custom colors for a new custom theme. The word `default` would be confusing here.